warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/edge/evaluation/EdgePreconditionJexlContext.java

```diff
@@ -108,17 +109,20 @@ private String cleanJexlTerm(String term) {
      */
     public void setFilteredContextForNormalizedContentInterface(Multimap<String,NormalizedContentInterface> nciEvent) {
         clearContext();
+        
         for (String filterFieldKey : filterFieldKeys) {
             Collection<NormalizedContentInterface> nciCollection = nciEvent.get(filterFieldKey);
             
             if (null != nciCollection) {
                 for (NormalizedContentInterface nci : nciCollection) {
+                    EventFieldValueTuple tuple = new EventFieldValueTuple();
                     
                     if (log.isTraceEnabled()) {
                         log.trace("Adding: " + filterFieldKey + "." + nci.getEventFieldValue() + " to context.");
                     }
-                    
-                    this.set(normalizeTerm(filterFieldKey), nci.getEventFieldValue());
+                    tuple.setFieldName(nci.getEventFieldName() == null ? nci.getIndexedFieldName() : nci.getEventFieldName());
+                    tuple.setValue(nci.getEventFieldValue());
```

Comment:
Should the event tuple use the normalized field value?  In fact it should probably contain both in case we have a jexl precondition that is attempting to match the unnormalized form.

Reply:
see comment above